docs/shrinkwrap.md

```diff
@@ -3,22 +3,37 @@ Shrinkwrap
 We use `npm shrinkwrap --dev` to lock down our dependency versions. This allows us to
 freeze all dependencies at the exact version we have installed in our node_modules.
 
-See [shrinkwrap docs](https://docs.npmjs.com/cli/shrinkwrap)
+See: [shrinkwrap docs](https://docs.npmjs.com/cli/shrinkwrap)
 
 ## Modifying Dependencies
 
-- In the root directory delete local node_modules
+We use clingwrap to avoid creating huge diffs in npm-shrinkwrap.json. clingwrap also removes
+`from` and `resolved` fields which is expected. If you happen to be pointing at a package
+that is not published with npm, you may want to update the npm-shrinkwrap.json by hand.
+
+- Install [clingwrap](https://github.com/goodeggs/clingwrap) globally: `npm install -g clingwrap`
+- Update your desired package, e.g. `npm install lodash@4.0.0 --save`
+- Clingwrap your updated package `clingwrap lodash`
+- Verify that Calypso works as expected and that tests pass.
+- Commit the updated package.json and npm-shrinkwrap.json
+
+## Modifying Sub-Dependencies
+
+Periodically, we'll want to bump sub-dependencies to pick up bugfixes. This may result
+in a large diff that is too big to review. It's very important that your node_modules 
+is deleted before you do this to be sure to pick up the latest versions.
```

Comment:
> This may result in a large diff that is too big to review.

Are you saying this is something for which we shouldn't expect a review? Or will this result in a large diff only if we do something wrong? Or is this just an assurance that if it results in a crazy huge diff, that can be expected?

> It's very important that your node_modules 
 is deleted before you do this to be sure to pick up the latest versions.

So by doing so we can only pick up all the latest subdependencies across our primary dependencies at once, right? What if we want to do this to pick up a fix in module A but we find an issue in a bump in module B?